### PR TITLE
Fix #107 : Reactivestack typings (cookies)

### LINKED ISSUE #107: Reactivestack typings
I would like to make a request for react-cookie to have typescript type definitions added. I want to use react-cookie in a project, but I need typings because I write all my React apps in tsx. Makes error catching much smoother.

### PR DESCRIPTION
**Pull request by Guymestef**
Hi,
I've created a first version of the typings.
Any modification needed? @eXon @PeterKottas

### CODE REVIEW

**Comment by PeterKottas** [comment]
Looking good as far as I can tell. Glad to be of help ;) I am using these atm for a real app and so far no problems.

File: packages/react-cookie/src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+declare module 'react-cookie' {
+    import * as React from 'react';
+
+    export type Cookie = string;
+
+    export class Cookies {
+        get: (key: string, options?: ReactCookieGetOptions) => Cookie;
```
Comment (FrancescoCioria) [inline packages/react-cookie/src/index.d.ts]: also, I think this should return `Cookie | undefined` as:

```js
new Cookies().get('foo') // => undefined
```

**Comment by eXon** [comment]
This is awesome! We might as well publish TS typing for universal-cookie `Cookies` class. I think someone posted it on the issue.

**Comment by eXon** [comment]
Oh I just saw it is included but within `react-cookie` only. Would it be possible to add it as well to the `universal-cookie` library?

**Comment by FrancescoCioria** [comment]
>  We might as well publish TS typing for universal-cookie Cookies class. I think someone posted it on the issue.

@eXon I think it's already been implemented: aren't [these ones](https://github.com/reactivestack/cookies/pull/124/files#diff-42d59860d8844b244c61c55f206b1285R6) the typings for the `Cookies` class?

**Comment by inunotaisho26** [comment]
@Guymestef Would be willing to help to. Could use the experience.

**Pull request merged**